### DISCUSSION TITLE
Request: Aliases

### DISCUSSION THREAD

**Discussion by Nerm335** [Zettlr/Zettlr#2570] [Ideas]
Hi there! 
I was wondering if it would be possible to have aliases like in obsidian or remnote. What I mean it's to give a second or third name to a page and use any of them to make a link. For example: a page is titled "Diabetes Mellitus type 2" and I give the alias "DM2". Whenever I want to make a link to the page I could use "Diabetes Mellitus type 2" or "DM2".
I am medical student looking at alternatives to obsidian and have found that zattlr has amazing features like the table editor, but I use the aliases quite a lot.
Thanks for the help.
PS: don't know if I'm posting in the right place, so I apologize in advance if it's the case.

**Comment by nathanlesage** [comment]
This could be a candidate for custom YAML frontmatter properties

**Reply by ThaddaeusWiedemer** [reply]
I think the question with this idea is also whether Zettlr could then support linking based on the alias, i.e., when typing `[[alias` Zettlr would propose the corresponding note in the tooltip and insert a link to `[[note]]`. If wiki links were a thing, it could even insert `[[note|alias]]`

**Reply by ThaddaeusWiedemer** [reply]
In Obsidian, there is a YAML frontmatter property of the form: `aliases: [alias1, alias2]`, which seems like a reasonable implementation.

**Comment by dzintars** [comment]
![image](https://user-images.githubusercontent.com/547420/179086971-47b4de01-8416-47de-a6db-85529005a54f.png)

Yeah... I just installed Zettlr and this was very first issue why i can't use it. All my notes are ID based and there is no way to link them (i don't know the ID of the note i am looking for). In Obsidian and ZK this is solved via frontmatter aliases or by parsing H1 or frontmatter `title:` property. I prefer Aliases as i can set multiple names for the same note. But i can live with `title` option as well.
Going back to [zk-nvim](https://github.com/mickael-menu/zk-nvim)

**Reply by nathanlesage** [reply]
You can activate a setting that will display h1 or title instead of filenames. I'm not sure the dropdown does this as well, if not, this is clearly a good feature request.

![image](https://user-images.githubusercontent.com/17251683/179168206-152e5094-b8c2-49fe-b527-d823e7f9193a.png)

**Comment by ghost** [comment]
Same issue here: I prefer Zettlr to Obsidian for Zettelkasten, but I also want to be able to work on mobile devices, for instance, to quickly add a note. I'm using timed IDs for filenames, so I need a title for Zettr and an alias for Obsidian. I can add both, but it's redundant.

As an FYI, There is an Obsidian plugin that lets me see the aliases in the file explorer, but the markdown heading plugin is broken. The only information Obsidian shows natively is the filename. Also, the alias: plugin doesn't yet work in graphs. Zettlr is the winner here by far.

Zettlr recognizes both tags: and keywords: in YAML—would it be possible to do the same for alias: and title:? Or releasing Zettlr for iOS would let me leave Obsidian behind entirely.  :-P  I'm pretty comfortable with Markdown and its variants (I've been using Ulysses for a while), but I'm just getting started with Zettlkasten.
